Assets/Scripts/UI/TitleScreen.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using UnityEngine;
+
+public class TitleScreen : MonoBehaviour {
+	[SerializeField] private MenuScreens mainMenu;
+	[SerializeField] private GameObject anyKeyPrompt;
+
+	private void Update() {
+		if (Input.anyKey) {
```

Comment:
Does this include clicking on a gamepad?

Reply:
I looked it up and indeed, this only works for keyboard and mouse. There seems to be no equivalent for gamepads in the built-in input system. I'm not quite sure how to go about making this work with gamepads yet. Any ideas?